scope_ascom.cpp

```diff
@@ -149,7 +149,6 @@ bool ScopeASCOM::Create(DispatchObj& obj)
         }
 
         Debug.Write(wxString::Format("Create ASCOM Scope: choice '%s' progid %s\n", m_choice, s_progid[m_choice]));
-

```

Comment:
cosmetic: can we not not remove this blank line -- I find it helpful to separate the Debug.Write noise from the actual code.